src/sql/workbench/browser/modelComponents/text.component.ts

```diff
@@ -26,14 +26,15 @@ import { IThemeService } from 'vs/platform/theme/common/themeService';
 	<div *ngIf="showDiv;else noDiv" style="display:flex;flex-flow:row;align-items:center;" [style.width]="getWidth()" [style.height]="getHeight()">
 	<p [title]="title" [ngStyle]="this.CSSStyles" [attr.role]="ariaRole" [attr.aria-hidden]="ariaHidden"></p>
```

Comment:
While I was looking at this I noticed that this seems kind of odd - it seems like this p element used to contain the text as its inner HTML but I moved that to the Link elements in https://github.com/microsoft/azuredatastudio/commit/44f4350df9b8ede64c8e975557706d1e16e539c3#diff-a58ae53937240196ff0c2e8b34229472710369499428b230a2a40324a3a69c8f and never cleaned this up. @alanrenmsft would you agree that the title should probably be added to the link element itself and change the p to just a div for styling purposes?